### PR TITLE
[LoongArch64-mono] Part-5:Add loongarch support in some files for LoongArch64

### PR DESCRIPTION
**Pull request by lawn123**
Add a new archecture LoongArch64 for mono.This PR is about loongarch support in some files.

### CODE REVIEW

**Comment by lawn123** [comment]
@shushanhf

File: src/mono/CMakeLists.txt [hunk 1]
```diff
@@ -392,6 +392,8 @@ elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "i686" OR CMAKE_SYSTEM_PROCESSOR STREQUAL
   set(CMAKE_SYSTEM_PROCESSOR "x86")
 elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "aarch64" OR CMAKE_SYSTEM_PROCESSOR STREQUAL "ARM64")
   set(CMAKE_SYSTEM_PROCESSOR "arm64")
+elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "loongarch64" OR CMAKE_SYSTEM_PROCESSOR STREQUAL "LOONGARCH64")
```
Comment (am11) [inline src/mono/CMakeLists.txt:395]: Also add https://github.com/dotnet/runtime/blob/4f9cd4d96e42ee36794480f026c8cad62b6a2407/src/mono/mono.proj#L457-L461 to enable LA64 mono cross-build.

File: src/mono/mono/mini/mini-arch.h [hunk 2]
```diff
@@ -21,6 +21,8 @@
 #include "mini-arm.h"
 #elif defined(TARGET_ARM64)
 #include "mini-arm64.h"
+#elif defined(__loongarch64)
```
Comment (am11) [inline src/mono/mono/mini/mini-arch.h:24]:
```suggestion
#elif defined(TARGET_LOONGARCH64)
```

File: src/tests/tracing/eventpipe/processinfo/processinfo.cs [hunk 3]
```diff
@@ -224,6 +224,7 @@ public static void TestEntryPoint()
                 Architecture.X64 => "x64",
                 Architecture.Arm => "arm32",
                 Architecture.Arm64 => "arm64",
+                Architecture.LoongArch64 => "loongarch64",
```
Comment (am11) [inline src/tests/tracing/eventpipe/processinfo/processinfo.cs:227]: nit: while at it, keep in sync:

```suggestion
                Architecture.LoongArch64 => "loongarch64",
                Architecture.RiscV64 => "loongarch64",
```

File: src/tests/tracing/eventpipe/processinfo2/processinfo2.cs [hunk 4]
```diff
@@ -225,6 +225,7 @@ public static void TestEntryPoint()
                 Architecture.X64 => "x64",
                 Architecture.Arm => "arm32",
                 Architecture.Arm64 => "arm64",
+                Architecture.LoongArch64 => "loongarch64",
```
Comment (am11) [inline src/tests/tracing/eventpipe/processinfo2/processinfo2.cs:228]:
```suggestion
                Architecture.LoongArch64 => "loongarch64",
                Architecture.RiscV64 => "riscv64",
```

File: src/tests/tracing/eventpipe/processinfo3/processinfo3.cs [hunk 5]
```diff
@@ -233,6 +233,7 @@ public static void TestEntryPoint()
                 Architecture.X64 => "x64",
                 Architecture.Arm => "arm32",
                 Architecture.Arm64 => "arm64",
+                Architecture.LoongArch64 => "loongarch64",
```
Comment (am11) [inline src/tests/tracing/eventpipe/processinfo3/processinfo3.cs:236]:
```suggestion
                Architecture.LoongArch64 => "loongarch64",
                Architecture.RiscV64 => "riscv64",
```

**Comment by lawn123** [comment]
@am11 Thank you very much. I have added the code you suggested to this PR.

**Pull request closed without merging**